lib/src/widgets/overlay.dart

```diff
@@ -143,14 +154,27 @@ class _DescribedFeatureOverlayState extends State<DescribedFeatureOverlay>
     _openController.dispose();
     _completeController.dispose();
     _dismissController.dispose();
-    _pulseController?.dispose();
+    _pulseController.dispose();
     super.dispose();
   }
 
+  @override
+  void didUpdateWidget(DescribedFeatureOverlay oldWidget) {
+    if (oldWidget.enablePulsingAnimation != widget.enablePulsingAnimation) {
+      if (widget.enablePulsingAnimation)
+        _pulseController.forward(from: 0);
+      else {
+        _pulseController.stop();
```

Comment:
This stops the animation abruptly. I think you should register a callback that stops the animation once it has completed.

Reply:
https://github.com/axel-op/feature_discovery/pull/35#discussion_r329324166